app/main.py

```diff
@@ -57,90 +49,10 @@ async def pdf_ocr(uuid: str):
             Bucket=os.getenv('BUCKET_NAME'),
             Key=f"{uuid}.pdf",
         )
-        fields = make_fields(response['Body'].read())
-        return {
-            "status": f"File received: {uuid}.pdf",
-            "body": fields,
-        }
+        fields = make_fields(uuid, response['Body'].read())
+        insert_case(fields)
+        return {"status": "Success"}
     except ConnectionError:
         return {"status": "Connection refused!"}
     except ClientError:
         return {"status": f"File not found: {uuid}.pdf"}
-
-
-@app.get("/visual/judges/{column_to_graph}/{judge_id}")
-async def vis_judges(column_to_graph : str, judge_id):
-    """
-    Queries judge_data from BE database, return ONE plotly express graph
-    """
-    db = os.getenv('DATABASE_URI')
-    query = f"SELECT {column_to_graph}, outcome FROM cases WHERE judge_id = {judge_id};"
-    
-    conn = psycopg2.connect(db, sslmode="require")
-    curs = conn.cursor()
-
-    df = pd.read_sql(query,conn)
-    conn.close()
-    return get_stacked_bar_chart(df, column_to_graph)
-
-
-@app.post("/vis/judges/")
```

Comment:
Were these endpoints meant to be deleted? We would be getting rid of the visualization endpoints, is that correct?

Reply:
Yes, the visualization endpoints are under development.

Reply:
Don't worry, they'll be back

Reply:
Understood, thank you for letting us know!